SampleApp/Warp/WarpApp.swift

```diff
@@ -0,0 +1,15 @@
+import SwiftUI
+import Warp_ios
+
+@main
+struct WarpApp: App {
+    init() {
+        JBMono.registerFonts()
```

Comment:
@vishkaischibsted didn't you remove this?

Reply:
It's in a removed app target.